flake.nix

```diff
@@ -0,0 +1,221 @@
+{
+  description = "BDK Flake to run all tests locally and in CI";
+
+  inputs = {
+    # stable nixpkgs (let's not YOLO on unstable)
+    nixpkgs.url = "github:NixOS/nixpkgs/nixos-23.11";
+
+    # pin dependencies to a specific version
+    # find instructions here:
+    # <https://lazamar.co.uk/nix-versions>
+
+    # bitcoind pinned to 0.25.1
+    nixpkgs-bitcoind.url = "github:nixos/nixpkgs?rev=53793ca0aecf67164c630ca34dbfde552a8ab085";
+    # TODO: pin to 0.26.0 once it stops to fail in darwin
+    # pinned to 0.26.0
+    # nixpkgs-bitcoind.url = "github:nixos/nixpkgs?rev=f5375ec98618347da6b036a5e06381ab7380db03";
+
+    # Blockstream's esplora
+    # inspired by fedimint CI
+    nixpkgs-kitman.url = "github:jkitman/nixpkgs?rev=61ccef8bc0a010a21ccdeb10a92220a47d8149ac";
```

Comment:
There maybe same issue with bitcoind.

Reply:
working on https://github.com/storopoli/bdk/pull/2

Reply:
Done in [96c1758](https://github.com/bitcoindevkit/bdk/pull/1320/commits/96c17585ec9f6aedcbf2efc8013619cd430fe7b4)